fontgenerator.js

```diff
@@ -144,7 +142,7 @@ TinyFontGenerator = {
     if(this.OS2data!==false) { return this.OS2data; }
 
     var data = "";
-    data += " 00 04";  // The current (Jan 2012) OS/2 table version is 4
+    data += " 00 01";  // The OS/2 table version is 1 (TrueType rev 1.66)
```

Comment:
in that case, you need to add the missing fields which are required by later versions of OS/2.